.github/workflows/kind-global-ci.yml

```
@@ -0,0 +1,463 @@
+          CYPRESS_git_password: "${{ secrets.GITHUB_TOKEN }}"
+          CYPRESS_git_key: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          working-directory: tackle-ui-tests
+          # spec can be removed once https://github.com/konveyor/tackle-ui-tests/pull/1331 merges
+          spec: "**/*.test.ts"
+
+      - name: Upload cypress report data as artifact
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: tackle-ui-test-reports-${{ inputs.test-tags }}
```

actionlint (1.7.8)

property "test-tags" is not defined in object type {api_hub_tests_ref: string; api_tests_ref: string; api_tests_tiers: string; component_name: string; operator_tag: string; run_api_tests: bool; run_ui_tests: bool; tag: string; ui_test_tags: string; ui_tests_ref: string}

(expression)